Raven.Studio.Html5/App/commands/database/cluster/extendRaftClusterCommand.ts

```diff
@@ -18,10 +18,6 @@ class extendRaftClusterCommand extends commandBase {
 
             var url = "/admin/cluster/join";
 
-            if (this.force) {
```

Comment:
What about removing force parameter: 
https://github.com/ravendb/ravendb/blob/v3.5/Raven.Studio.Html5/App/commands/database/cluster/extendRaftClusterCommand.ts#L7 ?
It is no longer used